src/robot/result/xmlelementhandlers.py

```diff
@@ -267,6 +267,15 @@ def start(self, elem, result):
         return result.body.create_break()
 
 
+@ElementHandler.register
+class ErrorHandler(ElementHandler):
+    tag = 'error'
+    children = frozenset(('status', 'msg', 'value'))
+
+    def start(self, elem, result):
+        return result.body.create_error()
```

Comment:
Is this enough to handle values? Are there tests for invalid syntax going through Rebot?

Reply:
Yes it is, they would not end in the log otherwise (or in the result model that is used in the tests).